packages/flutter/lib/src/gestures/tap.dart

```diff
@@ -267,5 +392,6 @@ class TapGestureRecognizer extends PrimaryPointerGestureRecognizer {
     properties.add(FlagProperty('wonArenaForPrimaryPointer', value: _wonArenaForPrimaryPointer, ifTrue: 'won arena'));
     properties.add(DiagnosticsProperty<Offset>('finalPosition', _finalPosition, defaultValue: null));
     properties.add(FlagProperty('sentTapDown', value: _sentTapDown, ifTrue: 'sent tap down'));
+    // TODO(tongmu): Add property _initialButtons and update related tests
```

Comment:
is there any reason we couldn't do this as part of this PR when it landed?

Reply:
It was because that this PR was already big and at its 2nd redo, so I didn’t want to introduce unnecessary changes that  affected a lot of files at that time. I’ll fix this.